app/assets/javascripts/html_editor.js

```diff
@@ -3,11 +3,15 @@
   App.HTMLEditor = {
     initialize: function() {
       $("textarea.html-area").each(function() {
+        var config = {
+          language: $("html").attr("lang"),
+          forcePasteAsPlainText: $(this).data("force-paste-plain-text") || false
```

Comment:
I remember soon after joining the project Rai mentioned "YAGNI". I feel exactly that way about this code: there's a good chance it'll stay here forever without anyone ever using it :thinking:. So I'd vote for removing the changes on this file and only add them if we ever need that option in the future.

Let me know if you disagree :wink:.

Reply:
My though was that probably it was deactivated for a reason and that probably there is some textarea inside the proposal dashboard where to have the "paste as plain text" is needed or desirable, but, of course, if we do not know which textarea is maybe is better to remove this change and wait for some user to complain. So i am going to remove it, at least we already know that we can configure this behavior on instances when we needed.

Reply:
Finally removed!